src/main/java/com/direwolf20/buildinggadgets/common/world_save/TemplateWorldSave.java

```diff
@@ -0,0 +1,77 @@
+package com.direwolf20.buildinggadgets.common.world_save;
+
+import com.direwolf20.buildinggadgets.BuildingGadgets;
+import com.direwolf20.buildinggadgets.common.schema.template.Template;
+import net.minecraft.nbt.CompoundNBT;
+import net.minecraft.nbt.INBT;
+import net.minecraft.nbt.ListNBT;
+import net.minecraft.world.server.ServerWorld;
+import net.minecraft.world.storage.WorldSavedData;
+import net.minecraftforge.common.util.Constants.NBT;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+public final class TemplateWorldSave extends WorldSavedData {
+    public static TemplateWorldSave getInstance(ServerWorld world) {
+        return null;//TODO: evaluate correct load Method
```

Comment:
Check the rewrite branch for a good implemenation of the worldsave system

Reply:
I know, I just didn't yet